content/docs/querying/basics.md

```diff
@@ -85,7 +85,8 @@ For example, this selects all `http_requests_total` time series for `staging`,
     http_requests_total{environment=~"staging|testing|development",method!="GET"}
 
 Label matchers that match empty label values also select all time series that do
-not have the specific label set at all. Regex-matches are fully anchored.
+not have the specific label set at all. Regex-matchers match partially and are not
```

Comment:
"match partially" isn't clear, does that mean that only some of my regex is applied? Stick to discussing anchoring.